tools/gen_internal_conf/src/main.rs

```diff
@@ -300,6 +300,52 @@ fn main() {
         );
         assert!(kernel_heap_max_size >= kernel_heap_init_size);
 
+        // Check validity of cache and disk size in mount options
+        const CACHE_PAGE_SIZE: usize = 0x1000;
+        const MIN_CACHE_SIZE: usize = 48 * CACHE_PAGE_SIZE; // 192KB
+        const MIN_DISK_SIZE: usize = 5 * 1024usize.pow(3); // 5GB
+        for mount in &occlum_config.mount {
+            if let Some(cache_size_str) = mount.options.cache_size.as_ref() {
+                let cache_size = {
+                    let cache_size = parse_memory_size(cache_size_str);
+                    if cache_size.is_err() {
+                        println!("The cache_size \"{}\" is not correct.", cache_size_str);
+                        return;
+                    }
+                    cache_size.unwrap()
+                };
+                if cache_size < MIN_CACHE_SIZE
+                    || cache_size % CACHE_PAGE_SIZE != 0
+                    || cache_size > kernel_heap_max_size
+                {
+                    println!(
+                        "Invalid cache_size \"{}\". The cache_size must be 1. larger than the minimum size \"{}\", \
+                        2. aligned with cache page size \"{}\", \
+                        3. smaller than the kernel_heap_max_size \"{}\".",
+                        cache_size, MIN_CACHE_SIZE, CACHE_PAGE_SIZE, kernel_heap_max_size
+                    );
+                    return;
+                }
+            }
+            if let Some(disk_size_str) = mount.options.disk_size.as_ref() {
```

Comment:
Need to check if it is Ext2Fs must set `disk_size`.

Reply:
Done